.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: '/'
+    schedule:
+      interval: monthly
+    open-pull-requests-limit: 5
```

Comment:
do you think there's anything useful we can get by enabling dependabot supports for the terraform ecosystem as well?  see https://docs.github.com/en/code-security/dependabot/ecosystems-supported-by-dependabot/supported-ecosystems-and-repositories

Reply:
While I don't recommend using dependabot's terraform support for customer projects, it should be very positive for public repositories like this terraform module. Let me add it to the setup.

Reply:
what do you think it will start to bump? are we using any "terraform" dependencies that enabling this will begin bumping?

Reply:
It will mainly take care of the terraform providers. As I'm using a terraform module in the example code, it should take care of it as well.